paper/src/main/java/vg/civcraft/mc/citadel/command/Activity.java

```diff
@@ -17,18 +19,18 @@ public class Activity extends BaseCommand {
 	public void execute(CommandSender sender) {
 		ActivityMapStat stat = Citadel.getInstance().getActivityMap().getStat();
 		if (stat == null) {
-			sender.sendMessage(ChatColor.RED + "ActivityMap is disabled");
+			sender.sendMessage(Component.text("ActivityMap is disabled").color(NamedTextColor.RED));
 			return;
 		}
 
-		sender.sendMessage(ChatColor.YELLOW + "[ActivityMap] object statistics:");
+		sender.sendMessage(Component.text("[ActivityMap] object statistics:").color(NamedTextColor.YELLOW));
 		sender.sendMessage("World count: " + stat.worlds);
 		sender.sendMessage("In-memory loaded regions: " + longToStr(stat.loadedRegions));
 		sender.sendMessage("In-memory NOT loaded regions: " + longToStr(stat.notLoadedRegions));
 		sender.sendMessage("In-memory groups: " + longToStr(stat.uniqueGroups));
 		sender.sendMessage("In-memory activities: " + longToStr(stat.loadedActivities));
 
-		sender.sendMessage(ChatColor.YELLOW + "[ActivityMap] region load statistics:");
+		sender.sendMessage(Component.text("[ActivityMap] region load statistics:").color(NamedTextColor.YELLOW));
```

Comment:
It's better to do `Component.text("Message", colour);`

What you're doing here is creating a new component with the text "[ActivityMap] region load statistics:", then creating another component that has that text but also the colour YELLOW. So you're creating two components here, not one.

Reply:
This function will be called like 0 - 4 times per day so I think doubled components are not a problem here.
I like a separate color more :)